TypeError: 'IPv4Address' object cannot be interpreted as an integer

After some pyton upgrade woes (now v3.10), I get the fllowing error:

```
Dec 23 18:40:11 twlz sqproxy[2022]: 2021-12-23 18:40:11,171 [INFO] [main] Building eBPF program ..., cflags=['-include', 'utils.h', '-DUSE_IPPORT_KEY']
Dec 23 18:40:14 twlz sqproxy[2022]: Traceback (most recent call last):
Dec 23 18:40:14 twlz sqproxy[2022]:   File "/usr/bin/sqredirect", line 8, in <module>
Dec 23 18:40:14 twlz sqproxy[2022]:     sys.exit(sqredirect())
Dec 23 18:40:14 twlz sqproxy[2022]:   File "/usr/lib/python3.10/site-packages/sqredirect/redirect.py", line 342, in sqredirect
Dec 23 18:40:14 twlz sqproxy[2022]:     main(args.ports, args.interface)
Dec 23 18:40:14 twlz sqproxy[2022]:   File "/usr/lib/python3.10/site-packages/sqredirect/redirect.py", line 163, in main
Dec 23 18:40:14 twlz sqproxy[2022]:     addr_gameserver2proxy_port[AddrKey(ip, game_port)] = c_uint16(proxy_port)
Dec 23 18:40:14 twlz sqproxy[2022]: TypeError: 'IPv4Address' object cannot be interpreted as an integer
Dec 23 18:40:14 twlz sqproxy[2011]: 2021-12-23 18:40:14,739 [ERROR] [source_query_proxy.epbf] eBPF redirection exit with code None
```

--- a/sqredirect/redirect.py
+++ b/sqredirect/redirect.py
@@ -160,8 +160,8 @@ def main(all_ports, interface=None):
 
         for ip, ports in all_ports.items():
             for game_port, proxy_port in ports.items():
-                addr_gameserver2proxy_port[AddrKey(ip, game_port)] = c_uint16(proxy_port)
-                addr_proxy2gameserver_port[AddrKey(ip, proxy_port)] = c_uint16(game_port)
+                addr_gameserver2proxy_port[AddrKey(int(ip), game_port)] = c_uint16(proxy_port)
+                addr_proxy2gameserver_port[AddrKey(int(ip), proxy_port)] = c_uint16(game_port)
 
     else:
         assert len(ip_addrs) == 1

